ui/src/message_popup/author/question_editing_components/learning_objectives.jsx

```diff
@@ -0,0 +1,54 @@
+/* @flow weak */
+import React from 'react';
+
+import {learningObjectives} from '../../../data/learning_objectives.js';
+
+import Paper from 'material-ui/Paper';
+import Divider from 'material-ui/Divider';
+import DropDownMenu from 'material-ui/DropDownMenu';
+import MenuItem from 'material-ui/MenuItem';
+
+export default React.createClass({
+  displayName: 'LearningObjectives',
+
+  propTypes: {
+    learningObjective: React.PropTypes.object.isRequired,
+    onChangeLearningObjective: React.PropTypes.func.isRequired
```

Comment:
Since this method is all about the learning objective changing, I'd suggest having this component handle the event from `DropDownMenu`, and then only calling `this.props.onChangeLearningObjective` with the `learningObjectiveId`.  That way the reference to the DOM event and other things that aren't important to the caller of this component are hidden.